swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing (Item):
+    def __init__ (self, category=str(), condition = 0):
+        self.category = "Clothing" 
+        self.condition = condition
```

Comment:
Here and in the other Item subclasses, the `category=str()` is not needed. You hardcode the category to "Clothing" on line 5, so the `category` that gets passed in as an argument is ignored, and thus unneeded.

Also, consider using `super` instead of copying the logic to set the category and condition. Even though it doesn't save you much here, it makes your code easier to extend if you ever add more complex logic to the Item constructor.